docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/event/Event.scss

```diff
@@ -0,0 +1,9 @@
+@charset "utf-8";
+@import "~Core/styles/variables";
+@import "~Core/styles/breakpoints";
+
+.event-location {
+    &> span {
```

Comment:
I don't believe you need to have the & operator here as the > child operator doesn't need to be appended directly to the parent to have the effect of limiting to one generation in the hierarchy. Have you tried it without?

Reply:
Yeah, the '&' is implied.  I'll remove it.